### DISCUSSION TITLE
Pls Approval of WalletConnect registry

### DISCUSSION THREAD

**Discussion by SnailMyth** [WalletConnect/walletconnect-monorepo#970] [Developer Support]
Hi there!
We have submitted an application to register our wallet "Hideout Wallet"
Would you be so kind as to approve the application so that our users can access WallectConnect as soon as possible.
Thanks in advance

**Comment by chadyj** [comment]
Responded.

**Reply by SnailMyth** [reply]
Thanks~

**Reply by SnailMyth** [reply]
@chadyj we had tried to summit the registry what named 'Hideout Wallet' agian.Could you please check this.Ttanks in advance.

**Reply by SnailMyth** [reply]
@chadyj My Project need to put on this to Google Pay,Please reply me when you get this notice as soon as possible. Please!Please!Please!
